### PR TITLE
Request to merge dev with master

### PR DESCRIPTION
**Pull request by Yurisas**
review plese

### CODE REVIEW

File: file1.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-row1
```
Comment (ysvirschuk) [inline file1.txt:1]: Not need to delete this row

File: first [hunk 2]
```diff
@@ -1,3 +1,5 @@
 The first file
 |@Yurisas Hi
 Well done !!!
+12345
+56789
```
Comment (ysvirschuk) [inline first:5]: OK-5

File: first [hunk 3]
```diff
@@ -1,3 +1,5 @@
 The first file
 |@Yurisas Hi
 Well done !!!
+12345
```
Comment (ysvirschuk) [inline first:4]: OK-4

File: first (hunk 2 above)
Comment (Yurisas) [reply first:5]: NOT OK!

File: first (hunk 3 above)
Comment (Yurisas) [reply first:4]: OK

File: file1.txt (hunk 1 above)
Comment (Yurisas) [reply file1.txt:1]: Agree.

**Pull request merged**